tests.js

```diff
@@ -54,6 +54,955 @@ window.Specs = {
 		}
 	},
 
+	"css2-box": {
+		"title": "CSS 2 Box Model",
+		"links": {
+			"tr": "CSS2/box.html",
+			"dev": "css2/"
+		},
+		"status": {
+			"stability": "stable",
+			"first-snapshot": 1998,
+			"last-snapshot": 1998
+		},
+		"properties": {
+			"border-color": {
+				"links": {
+					"tr": "#propdef-border-color",
```

Comment:
Why is this anchor different?

Reply:
That was a mistake. In one or two cases the anchor was different in the TR vs. the drafts version. Here it's the same, so I've made them the same.

We should also generally discuss whether we prefer linking to the headline or the property definition. For now, I've preferably linked to the headline but linked to the definition if the headline was too far apart from the definiton.

Reply:
I see benefits to link directly to the property definition, at least for all the spec made with Bikeshed: closer to the grammar (that's what we test), likely more resilient than sections headlines to editorial changes of the spec. We could even generate some of them instead of filling them in for each feature.

But currently we use to link to headlines, so let's discuss changing that in another issue.